src/views/document/utils/boxes/CommentsBox.vue

```diff
@@ -255,4 +255,9 @@
     }
   }
 
+  .button{
```

Comment:
Put this class inside a new style tag, and remove scoped from the original style tag